tests/testthat/_snaps/deploymentTarget.md

```diff
@@ -3,7 +3,7 @@
     Code
       deploymentTarget()
     Condition
-      Error in `deploymentTarget()`:
+      Error in `accountInfo()`:
```

Comment:
This indicates that `caller_env` probably isn't being passed correctly.

Reply:
Yes, absolutely. The problem is that `accountInfo()` is part of the public API and does not accept `caller_env`. Previously, we were calling `findAccount()` in some contexts. That does take `caller_env`, but does not return the full account detail. I wanted to avoid reasoning about what shape of account information was available, so I switched to always calling `accountInfo()`. The drawback is that we lose `caller_env`.

How would you resolve this complication?

Reply:
I'd be tempted to push more of the logic out of `accountInfo()` and into `findAccount()`, then switch back to calling `findAccount()` everywhere. Then `accountInfo()` just becomes a very thin wrapper around `findAccount()`.

Alternatively, you could accept that `findAccount()` is mostly an internal facing function that users are unlikely to call directly, and just add an `error_call` argument to it.

Reply:
I've created `findAccountInfo()` as a version of `accountInfo()` that accepts error context and lives alongside `findAccount()`, which continues to identify the target account name+server.